testing/tests/DevExpress.ui.widgets.dataGrid/filterRow.tests.js

```diff
@@ -74,6 +74,7 @@ QUnit.module('Filter Row', {
         fx.off = true;
     },
     afterEach: function() {
+        this.dispose();
```

Comment:
Is this change from another PR?

Reply:
No, I have found problems with the filter row tests in current PR and fixed them as part of this PR.